appveyor.yml

```diff
@@ -0,0 +1,140 @@
+version: 1.0.{build}
```

Comment:
@sergey-shandar at this point we are not planning to enable appveyor.
Can you add this under tools\autogetnForSwaggers and not in the root?